next/frontend/utils/general.ts

```diff
@@ -105,3 +107,20 @@ export const downloadBlob = (blob: Blob, fileName: string) => {
 export function isDefined<T>(value: T | undefined | null): value is T {
   return value !== undefined && value !== null
 }
+export const getTitle = (
```

Comment:
I would rename it to `getFormTitle` and move from general utils to something form-specific.